packages/pwa/app/utils/routes-utils.js

```diff
@@ -7,6 +7,27 @@
 
 import {getSites} from './site-utils'
 import {urlPartPositions} from '../constants'
+import {matchPath} from 'react-router-dom'
+
+/**
+ * Check if a path matches an array of routes.
+ * @param path path that
+ * @param routes
+ * @returns {*}
+ */
+export const matchRoute = (path, routes) => {
```

Comment:
Chatted with Adam, we are looking at moving this into the SDK.

Reply:
Done in a related PR: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/478